initdata-processor/policy/policy.go

```diff
@@ -0,0 +1,56 @@
+package policy
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+
+	_ "embed"
+)
+
+//go:embed assets/deny-with-message.rego
+var denyWithMessagePolicy []byte
+
+// DenyWithMessage returns a Rego policy that answers all Kata agent requests with the given message.
+func DenyWithMessage(format string, params ...any) []byte {
+	buf := &bytes.Buffer{}
+	buf.Write(denyWithMessagePolicy)
+	buf.WriteString("\n\nmessage := ")
+	fmt.Fprint(buf, regoEscape(fmt.Sprintf(format, params...)))
+	return buf.Bytes()
+}
+
+// regoEscape returns a double-quoted Rego literal for s.
```

Comment:
Is there any source for this that states who strings should be escaped in rego?

Another way might be to call json.Marshal on the string?